src/main.rs

```diff
@@ -289,6 +292,7 @@ pub struct Args {
     no_default_features: bool,
     target: Option<String>,
     target_dir: Option<String>,
+    rustlib_target: Option<String>,
```

Comment:
What is this flag?

Reply:
If we use custom target, we use `--target` like this : `--target foo.json`.
And we can specify the [llvm-target](https://docs.rust-embedded.org/embedonomicon/custom-target.html#fill-the-target-file) in json.
So, the rustlib_target means llvm-target. (e.g. aarch64-unknown-none)

Reply:
Does cargo itself has this flag?

Reply:
I guess not.

Reply:
Then we have to skip it. `cargo bloat` mimics `cargo build` commands.

Reply:
I see. I'll find another way then. 
Thank you for the review.